modyn/models/modular_adapters/modular_adapters.py

```diff
@@ -0,0 +1,36 @@
+from peft import LoraConfig, TaskType, get_peft_model
```

Comment:
Please add some docstrings on how (and where) those  functions are supposed to be used